src/vm/runners/builtin_runner/ec_op.rs

```diff
@@ -492,190 +482,158 @@ mod tests {
-    fn compute_ec_op_invalid_same_x_coordinate() {
-        let partial_sum = (bigint!(1), bigint!(9));
-        let doubled_point = (bigint!(1), bigint!(12));
-        let m = bigint!(34);
-        let alpha = bigint!(1);
-        let height = 256;
-        let prime = bigint_str!(
-            b"3618502788666131213697322783095070105623107215331596699973092056135872020481"
-        );
-        let result = EcOpBuiltinRunner::ec_op_impl(
-            partial_sum.clone(),
-            doubled_point.clone(),
-            &m,
-            &alpha,
-            &prime,
-            height,
-        );
-        assert_eq!(
-            result,
-            Err(RunnerError::EcOpSameXCoordinate(
-                partial_sum,
-                m,
-                doubled_point
-            ))
-        );
-    }
+        #[test]
+        fn compute_ec_op_impl_valid_b() {
+            let partial_sum = (
+                felt_str!(
+                    "2962412995502985605007699495352191122971573493113767820301112397466445942584"
+                ),
+                felt_str!(
+                    "214950771763870898744428659242275426967582168179217139798831865603966154129"
+                ),
+            );
+            let doubled_point = (
+                felt_str!(
+                    "874739451078007766457464989774322083649278607533249481151382481072868806602"
+                ),
+                felt_str!(
+                    "152666792071518830868575557812948353041420400780739481342941381225525861407"
+                ),
+            );
+            let m = Felt::new(34);
+            let alpha = Felt::one();
+            let height = 256;
+            let result = EcOpBuiltinRunner::ec_op_impl(partial_sum, doubled_point, &m, &alpha, height);
+            assert_eq!(
+                result,
+                Ok((
+                    felt_str!(
+                        "2778063437308421278851140253538604815869848682781135193774472480292420096757"
+                    ),
+                    felt_str!(
+                        "3598390311618116577316045819420613574162151407434885460365915347732568210029"
+                    )
+                ))
+            );
+        }
 
+        #[test]
+        fn compute_ec_op_invalid_same_x_coordinate() {
+            let partial_sum = (Felt::one(), Felt::new(9));
+            let doubled_point = (Felt::one(), Felt::new(12));
+            let m = Felt::new(34);
+            let alpha = Felt::one();
+            let height = 256;
+            let result = EcOpBuiltinRunner::ec_op_impl(
+                partial_sum.clone(),
+                doubled_point.clone(),
+                &m,
+                &alpha,
+                height,
+            );
+            assert_eq!(
+                result,
+                Err(RunnerError::EcOpSameXCoordinate(
+                    partial_sum,
+                    m,
+                    doubled_point
+                ))
+            );
+        }
+    */
```

Comment:
Uncomment the tests.

Reply:
Resolved: https://github.com/lambdaclass/cairo-rs/pull/630/commits/d69b461fa7d7b0b8d9004566d5b28f1aaac5ea8e